spark/src/test/scala/io/delta/sharing/spark/DeltaSharingFileProfileProviderSuite.scala

```diff
@@ -70,8 +70,8 @@ class DeltaSharingFileProfileProviderSuite extends SparkFunSuite {
     )
   }
 
-  test("version is missing") {
-    val e = intercept[IllegalArgumentException] {
+  test("shareCredentialsVersion is missing") {
+    val e = intercept[Exception] {
```

Comment:
Did the type of exception change?

Reply:
Yes, it changes with the new scala version.